src/app/ui/entries_list/mod.rs

```diff
@@ -222,6 +222,7 @@ impl<'a> EntriesList {
         area: Rect,
         list_keymaps: &[Keymap],
         has_filter: bool,
+        entries_len: usize,
```

Comment:
I don't think we need to render the entries count with the place holder since we don't have entries anymore. I would rather, remove the argument from here and make the items_count in the function `get_list_block(...)` optional. More details are provided in the code snippet there as well.